R/model.R

```diff
@@ -119,6 +119,7 @@ run_resumable_simulation <- function(
   events <- create_events(parameters)
   initialise_events(events, variables, parameters)
   renderer <- individual::Render$new(timesteps)
+  populate_incidence_rendering_columns(renderer, parameters)
```

Comment:
Can we add this to `run_simulation` and `run_metapop_simulation` too?

Reply:
So I think run_simulation calls the run_resumable_simulation anyway, so the incidence populate_incidence_rendering_columns should be called there. I've added a wrapper version that loops round the list of rendering objects to the run_metapop_simulation function which I think is working.